.github/workflows/release_build_test.yaml

```diff
@@ -70,15 +70,18 @@ jobs:
       build_type: ${{ github.event.inputs.build_type || 'Release' }}
     secrets: inherit
 
-  Package:
-    if: github.ref_type == 'tag'
+  PackageAndUpload:
+    # if: github.ref_type == 'tag'
```

Comment:
Why have the comment here (uncomment & make it work or delete)?

Reply:
Commented that to test the workflow [here](https://github.com/memgraph/memgraph/actions/runs/8468146539). When it passes I will uncomment this :+1: